Fixes #1173: Change Telescope theme to a light theme

hyunjiLeeTech opened the pull request:
## Issue This PR Addresses

#1173. Change Telescope Theme to a light theme

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Homepage
![image](https://user-images.githubusercontent.com/45149474/98846121-a7eddd00-241c-11eb-9d33-4078dfae8467.png)

About page
![image](https://user-images.githubusercontent.com/45149474/98846163-b2a87200-241c-11eb-98b1-3c5e928e11cd.png)

My feed page
![image](https://user-images.githubusercontent.com/45149474/98846207-c0f68e00-241c-11eb-8ee8-d40d29d0703b.png)

404 page
![image](https://user-images.githubusercontent.com/45149474/98846472-0adf7400-241d-11eb-9363-b9e34944aa8d.png)

While I'am working on lighten the 404 page, I've found the homepage, about page, and my feed page also got lighter theme following my change. I've changed background and primary colour on 'theme.js' file. Please have a look at this colour change is whether what you or not and let me know if you want another colour theme.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

agarcia-caicedo reviewed (changes requested):
Hi, to turn convert the 404 page to light mode, you shouldn't change the theme.js file, that changes everything that uses that file. Hence why all the other pages got changed. We already decided on the colours in pr #1174

agarcia-caicedo reviewed (changes requested):
In the error.jsx file, can you change the color of the font for the "let's go back" button, so that it uses the dark grey instead of the off-white? The contrast isn't great with the white

hyunjiLeeTech commented:
Hi, sorry about the changes of the background and default colour. I've changed background/default colour to original colour you've decided (#e5e5e5) and changed the "let's go back" button colour to "#d3d3d3". Please let me know if you need another change or want another colour. Thank you.

agarcia-caicedo commented on src/frontend/src/pages/error.jsx:
```diff
@@ -72,7 +72,7 @@ const useStyles = makeStyles((theme) => ({
     position: 'relative',
     fontSize: '1.5rem',
-    color: 'white',
+    color: '#D3D3D3',
```
Sorry, I should have been clearer. This "color: theme.palette.text.default" is what I meant.

hyunjiLeeTech commented:
I've change the font colour to "theme.pallette.text.default". This is how it looks like now :). Please let me know if you want something more about it. Thanks.
![image](https://user-images.githubusercontent.com/45149474/99007255-d699af80-2511-11eb-9ebe-55df1e5c1b8a.png)

agarcia-caicedo reviewed (approved):
Thanks for the contribution!
